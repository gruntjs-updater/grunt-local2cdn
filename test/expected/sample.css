body {
        background-image: url('http://cdn.bootcss.com/xxx/foo.jpg');
        background-image: url("http://cdn.bootcss.com/xxx/foo.jpg");
        background-image: url(http://cdn.bootcss.com/xxx/foo.jpg);
        background-image: url('http://cdn.bootcss.com/xxx/foo.png');
        background-image: url('http://cdn.bootcss.com/xxx/foo.png');
        
        background-image: url('http://demo.com/bar/foo.png');
        background-image: url('http://cdn.bootcss.com/xxx/foo.gif');
        background-image: url('foo/bar.jpg');
        background-image: url('data:gif; base64XQTRTWEQW');
}
