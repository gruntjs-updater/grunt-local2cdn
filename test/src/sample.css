body {
        background-image: url('foo.jpg');
        background-image: url("foo.jpg");
        background-image: url(foo.jpg);
        background-image: url('/foo.png');
        background-image: url('bar/foo.png');
        
        background-image: url('http://demo.com/bar/foo.png');
        background-image: url('../foo.gif');
        background-image: url('foo/bar.jpg');
        background-image: url('data:gif; base64XQTRTWEQW');
}
