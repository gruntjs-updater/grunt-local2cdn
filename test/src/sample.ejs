<%- include top %>
<style>
      body {
        background-image: url('foo.jpg');
        background-image: url("foo.jpg");
        background-image: url(foo.jpg);
        background-image: url('/foo.png');
        background-image: url('bar/foo.png');
        
        background-image: url('http://demo.com/bar/foo.png');
        background-image: url('../foo.gif');
        background-image: url('foo/bar.jpg');
        background-image: url('data:gif; base64XQTRTWEQW');
      }
    </style>
    <!-- Bootstrap Core-->
    <link href="/libs/bootstrap/css/bootstrap.css" rel="stylesheet">
  </head>
  
  <body>
    <h1>Hello</h1>
    <img something src="foo.jpg" data-hiya data-food="crimson">

    <img src=foo/lala.png yo="what" />
    <%
        var str = '<img src="foo.png" yo="what" />';
    %>
    <img src="/foo.gif" something=blah  > :)
    <img src="http://demo.com/bar/foo.png" something=blah  >
    <img src="bar/foo.png" something=blah  >

    <!-- jquery.js -->
    <script src="/libs/jquery.js"></script>
	<script src="/libs/bootstrap/js/bootstrap.js"></script>
<%- include bottom %>
