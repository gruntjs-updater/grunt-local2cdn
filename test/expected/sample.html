<html>
  <head>
    <style>
      body {
        background-image: url('http://cdn.bootcss.com/xxx/foo.jpg');
        background-image: url("http://cdn.bootcss.com/xxx/foo.jpg");
        background-image: url(http://cdn.bootcss.com/xxx/foo.jpg);
        background-image: url('http://cdn.bootcss.com/xxx/foo.png');
        background-image: url('http://cdn.bootcss.com/xxx/foo.png');
        
        background-image: url('http://demo.com/bar/foo.png');
        background-image: url('http://cdn.bootcss.com/xxx/foo.gif');
        background-image: url('foo/bar.jpg');
        background-image: url('data:gif; base64XQTRTWEQW');
      }
    </style>
    <!-- Bootstrap Core-->
    <link href="http://cdn.bootcss.com/bootstrap/3.1.1/css/bootstrap.min.css" rel="stylesheet">
  </head>
  
  <body>
    <h1>Hello</h1>
    <img something src="http://cdn.bootcss.com/xxx/foo.jpg" data-hiya data-food="crimson">

    <img src=foo/lala.png yo="what" />

    <img src="http://cdn.bootcss.com/xxx/foo.gif" something=blah  > :)
    <img src="http://demo.com/bar/foo.png" something=blah  >
    <img src="http://cdn.bootcss.com/xxx/foo.png" something=blah  >

    
    <img src="http://cdn.bootcss.com/xxx/foo.tif" />

    <!-- jquery.js -->
    <script src="http://cdn.bootcss.com/jquery/2.0.3/jquery.min.js"></script>
	<script src="http://cdn.bootcss.com/bootstrap/3.1.1/js/bootstrap.min.js"></script>
  </body>
</html>
